airflow/auth/managers/simple/simple_auth_manager.py

```diff
@@ -78,20 +80,23 @@ class SimpleAuthManager(BaseAuthManager):
     :param appbuilder: the flask app builder
     """
 
-    # File that contains the generated passwords
-    GENERATED_PASSWORDS_FILE = (
-        AIRFLOW_ROOT_PATH / "generated" / "simple_auth_manager_passwords.json.generated"
-    )
-
     # Cache containing the password associated to a username
     passwords: dict[str, str] = {}
 
+    @cached_property
+    def generated_password_file(self) -> Path:
+        filename = "simple_auth_manager_passwords.json.generated"
+        if os.getenv("BREEZE", False):
+            return Path(f"/files/{filename}")
```

Comment:
It feels weird to have Breeze-specific logic in the code

Reply:
Agree but I could not find a better way to do it. The `files` directory is basically in two different locations depending on whether you run Airflow with Breeze